scripts/variantstore/wdl/GvsAssignIds.wdl

```diff
@@ -13,11 +13,15 @@ workflow GvsAssignIds {
     Boolean samples_are_controls = false
 
     File? assign_ids_gatk_override
+    Boolean parse_vcf_headers = false
   }
 
   String sample_info_table = "sample_info"
   String sample_info_schema_json = '[{"name": "sample_name","type": "STRING","mode": "REQUIRED"},{"name": "sample_id","type": "INTEGER","mode": "NULLABLE"},{"name":"is_loaded","type":"BOOLEAN","mode":"NULLABLE"},{"name":"is_control","type":"BOOLEAN","mode":"REQUIRED"},{"name":"withdrawn","type":"TIMESTAMP","mode":"NULLABLE"}]'
-  String sample_load_status_json = '[{"name": "sample_id","type": "INTEGER","mode": "REQUIRED"},{"name":"status","type":"STRING","mode":"REQUIRED"}, {"name":"event_timestamp","type":"TIMESTAMP","mode":"REQUIRED"}]'
+  String vcf_header_lines_temp_schema_json = '[{"name": "sample_id","type": "INTEGER","mode": "REQUIRED"},{"name":"vcf_header_lines","type":"STRING","mode":"NULLABLE"},{"name":"vcf_header_lines_hash","type":"STRING","mode":"REQUIRED"}]'
```

Comment:
I'm not seeing a line number associated with the header here or for `sample_vcf_header`; do we really not care about capturing that info?

Reply:
I don't think the order of the header lines is significant.